pkg/codegen/python/gen.go

```diff
@@ -2556,6 +2583,35 @@ func (mod *modContext) genType(w io.Writer, name, comment string, properties []*
 	}
 	fmt.Fprintf(w, "):\n")
 	mod.genTypeDocstring(w, comment, props)
+	if len(props) == 0 {
+		fmt.Fprintf(w, "        pass\n")
+	} else {
+		fmt.Fprintf(w, "        %s.__configure__(\n", name)
+		for _, prop := range props {
+			pname := PyName(prop.Name)
+			fmt.Fprintf(w, "            %[1]s=%[1]s,\n", pname)
+		}
+		fmt.Fprintf(w, "            _setter=lambda key, value: pulumi.set(__self__, key, value),\n")
+		fmt.Fprintf(w, "        )\n")
+	}
+
+	// Define method signature
+	fmt.Fprintf(w, "    @staticmethod\n")
+	fmt.Fprintf(w, "    def __configure__(*,")
```

Comment:
[PEP8](https://peps.python.org/pep-0008/#descriptive-naming-styles) this should just be `__configure`.

Reply:
I need to call `ResourceArgs.__configure__` in the `Resource` class. Any suggestions

Reply:
That is also emit'd by SDK gen so just change that as well.